### PR TITLE
Feature/desktop version

### PR DESCRIPTION
**Pull request by GerardoGomezG**
# Desktop version ready.

Here's a snapshot:

imagen

### CODE REVIEW

**Review by danyhoshi — CHANGES_REQUESTED** [review]
# Fantastic work @GerardoGomezG !!! 🥇 

# Your code is very clean! 👍 

## I only suggest a few changes for coming to a better finish 
### I recommend removing this horizontal scrolling
horizontal-scrolling

### I propose for this section place the form centering and conserve together these elements (like the original design for these elements) 



### Original design
228373310-597574e4-d500-4d2f-8d7c-29e367c3c4e6

# You can do it! 💪

File: styles.css [hunk 1]
```diff
@@ -249,49 +293,184 @@ input:-webkit-autofill:active {
...
+    margin: 0 auto;
+    height: auto;
+    max-width: 500px;
+    width: 45vw;
+  }
+
+  #about-me {
+    display: inline-flex;
+    margin: 4em 0;
+    width: 100vw;
```
[the remark this answers is not in the document]
Comment (GerardoGomezG) [reply styles.css]: Hello @danyhoshi !

Thank you for taking time to review my work

All your suggestions have been applied!

Screenshot 2023-04-06 at 17-37-50 Gerardo's Portfolio

[the remark this answers is not in the document]
Comment (danyhoshi) [reply styles.css]: # Great! 👍 😄

**Review by danyhoshi — APPROVED** [review]
# Great job!!! 😄 
Time to merge!!!

**Review by danyhoshi — APPROVED** [review]
# Ready to merge!!! 😄

**Pull request merged**